### PR TITLE
adding jokes support

### PR DESCRIPTION
**Pull request by houssem-yahiaoui**
Adding minimal jokes support as mentioned in issue #19 .

### CODE REVIEW

File: src/lib/constants.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ const JOBS = [
   /\b(kh(e|a|)d(e|)ma|jobs?|boulots?)\b/i,
 ]
 
+const JOKES = [
+  /\b((e|a)(7|h)k(i|)li|jokes?|d(a|)(7|h)akni|(7|h)ajini)\b/i,
```
Comment (youknowriad) [inline src/lib/constants.js]: j'aurai rajouté : rire, blague, laught

Comment (houssem-yahiaoui) [reply src/lib/constants.js]: Okay, je vais les rajouter.

**Comment by houssem-yahiaoui** [comment]
@youknowriad normally it's done.

**Review by youknowriad — APPROVED** [review]
Tks 👍

**Pull request merged**